Add Hextris Game

kartikjoshi267 opened the pull request:
## PR Description 📜
Added Hextris Game
Please review the PR!

Fixes #1901 

<hr>
 
## Mark the task you have completed ✅

<!----Please delete options that are not relevant. In order to tick the check box just but x inside them for example [x] like this----->

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have specified the respective issue number for which I have requested the new game.


<hr>

## Add your screenshots(Optional) 📸
![image](https://github.com/kunjgit/GameZone/assets/96565730/26abd7ea-0550-47ce-af5d-f8d1cfb67c04)




--- 
<br>

## Thank you soo much for contributing to our repository 💗

kunjgit commented:
Thank you @kartikjoshi267 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

kunjgit reviewed (changes requested):
Hello @kartikjoshi267,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

kunjgit reviewed (changes requested):
Hello @kartikjoshi267,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

kunjgit reviewed (changes requested):
Hello @kartikjoshi267,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

kartikjoshi267 commented:
> Hello @kartikjoshi267, You need to make the following changes:
> 
> * Please modify the main README.md file.
> * Please add an image to the assets/images directory.
> 
> Hoping that you will make those changes soon 🚀

@kunjgit I have already added an image to assets/image directory and also modified main README.md file. Please review the PR again.

kunjgit reviewed (changes requested):
Hello @kartikjoshi267,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

kartikjoshi267 commented:
@kunjgit Both completed

kunjgit commented:
please make sure you are not adding game as before you also there are many games which are already accepted and it is pending to add so don't do that changes !!! 👀

kunjgit reviewed (changes requested):
Hello @kartikjoshi267,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

kartikjoshi267 commented:
> please make sure you are not adding game as before you also there are many games which are already accepted and it is pending to add so don't do that changes !!! 👀

@kunjgit Please check now!

kunjgit commented:
Thank you  @kartikjoshi267 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

kunjgit commented:
Hey @kartikjoshi267,
 You can't work on a closed issue 👀! 
 Follow the proper guideline and make a new PR again 😀. 
 Happy Hacking 💗
